src/main/java/lottoautomatic/domain/LottoProfit.java

```diff
@@ -0,0 +1,39 @@
+package lottoautomatic.domain;
+
+import java.util.Arrays;
+
+public enum LottoProfit {
+
+	THREE(3, 5000),
+	FOUR(4, 50000),
+	FIVE(5, 1500000),
+	SIX(6, 2000000000);
```

Comment:
```suggestion
	THREE(3, 5_000),
	FOUR(4, 50_000),
	FIVE(5, 1_500_000),
	SIX(6, 2_000_000_000);
```
이렇게 표현하면 가독성에 더 좋답니다!